src/theme/Footer/index.js

```diff
@@ -82,8 +82,16 @@ function Footer() {
         dangerouslySetInnerHTML={{
           __html: copyright
         }} /> : null}
+        <div className="footer__legal" >
+          <ul>
+            <li><a href="https://unity.com/legal">Legal</a></li>
+            <li><a href="https://unity.com/legal/privacy-policy">Privacy Policy </a></li>
+            <li><a href="https://unity.com/legal/cookie-policy">Cookies</a></li>
+            <li><a href="https://unity.com/legal/do-not-sell-my-personal-information">Do Not Sell or Share My Personal Information</a></li>
+         </ul>
+        </div>
         <div className="footer__details" >"Unity", Unity logos, and other Unity trademarks are trademarks or registered trademarks of Unity Technologies or its affiliates in the U.S. and elsewhere (<a href="https://unity3d.com/legal/trademarks">more info here</a>). Other names or brands are trademarks of their respective owners.</div>
-          </div>}
+        </div>}
       </div>
       <script type="text/javascript">
         function OptanonWrapper() { }
```

Comment:
This is a helper function used by OneTrust. It should have been defined in the `<head>` section of the page, right after the OneTrust SDK loader script.

Please refer to [#talk-onetrust](https://unity.slack.com/archives/C056B1X1H6D) channel, and the channel bookmarks for "Web Tracking Implementation" document, and the Integration Demo for the simplest HTML example.

Reply:
@gormus I don't understand this part. Could you please tell me what this means and what to put where? Thanks so much for all of your help with this. We appreciate it!

Reply:
@amanda-butler-unity I believe [this is the relevant section](https://docs.google.com/document/d/1takg_GmIBBKKTj-GHZCwzxohpQz7Bhekivkk72kYMtE/edit#heading=h.9izhrgvd9dot).
What I can't see from eyeballing the docs-multiplayer code is an obvious place to specify custom `<head></head>` contents that will be applied to every page, where you would insert the snippet from the google doc.
There doesn't appear to be an equivalent to the footer customization functions in src/theme/Footer/index.js for headers.
You may need a developer familiar with how Docusaurus works to add an entry point like that.

Reply:
Revisited this.
@gormus the OneTrust include is provided already by this https://github.com/Unity-Technologies/com.unity.multiplayer.docs/blob/main/config/scripts.config.js and the ID appears to be correct for *.unity3d.com domains.

Reply:
The header does not comply with the prescribed order in the doc, and the `scripts.config.js` alludes to this:
> //todo: GTM implementation that relates to OT and user data tracking

On `view-source:https://docs-multiplayer.unity3d.com/` Google Tag Manager precedes OneTrust and I can't find anything relating to the dataLayer setting, so one thing is missing and the others are in the wrong order.

Should that block this PR from landing? It's true in the main branch.

Reply:
@antoniancu My team is happy to add whatever needs to be added. Just let us know what to put where :)

Reply:
I understand that we'd normally put this in the <head> however since this is a docusaurus site (with a lot of automagical stuff) that's not something I'm sure we can do (or at least I haven't figured out how to yet). 

The Script itself appears to work correctly in its current location in the page - is this a blocker @gormus ?

Reply:
Sorry, I didn't mean to be a blocker :) 

I couldn't find any preview URL for the PR, so I compared with https://docs-multiplayer.unity3d.com and left a review based on that. I can't be sure if applies to this PR :)